guides/user/canary/config/filter_templates/index.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: single
+title:  "Create and use filter templates"
+sidebar:
+  nav: guides
+---
+
+{% include toc %}
+
+You can create sets of filters to apply against time series returned by your
```

Comment:
"apply against time series returned by your telemetry provider" reads a little off, since filter templates apply to queries, but not to time series (which are the responses to queries).

Reply:
I think it might be best to describe filter templates by providing examples for each telemetry provider, but you might say something like "filter templates allow you to compose and parameterize advanced queries against your telemetry provider" and what that amounts to is going to be slightly different for each provider, since the query syntax will vary for each.

Reply:
That's good wording. Getting quality examples for each provider (or at least two) can be an enhancement, later.